packages/camera/camera/lib/src/camera_image.dart

```diff
@@ -106,6 +116,37 @@ ImageFormatGroup _asImageFormatGroup(dynamic rawFormat) {
   return ImageFormatGroup.unknown;
 }
 
+/// Converts [ImageFormatGroup] to integer definition of the raw format
+int imageFormatGroupAsIntegerValue(ImageFormatGroup imageFormatGroup) {
+  if (defaultTargetPlatform == TargetPlatform.iOS) {
+    switch (imageFormatGroup) {
+      // kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange
+      case ImageFormatGroup.yuv420:
+        return 875704438;
+      // kCVPixelFormatType_32BGRA
+      case ImageFormatGroup.bgra8888:
+        return 1111970369;
+      case ImageFormatGroup.jpeg:
+      case ImageFormatGroup.unknown:
+        return 0;
+    }
+  } else if (defaultTargetPlatform == TargetPlatform.android) {
+    switch (imageFormatGroup) {
+    // kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange
+      case ImageFormatGroup.yuv420:
+        return 35;
+    // kCVPixelFormatType_32BGRA
+      case ImageFormatGroup.bgra8888:
+      case ImageFormatGroup.unknown:
+        return 0;
+      case ImageFormatGroup.jpeg:
+        return 256;
+    }
+  }
+  // unknown ImageFormatGroup or unsupported platform
+  return 0;
+}
+
```

Comment:
I would move this method into the `Utils` class. The logic inside the `camera_image.dart` file is only used by the `startImageStream` and `stopImageStream` methods, which in the future will most likely be replaced. I can image the `imageFormatGroup` is also interesting for taking a picture or recording videos. 

The `Utils` class already contains a similar function, so this would be a nice place.

Reply:
Not sure if I would agree. On the one hand, yes. This could be placed in a separate class. 
On the other hand, the conversion the other way around is found in this class as a private method. `_asImageFormatGroup`. But we could move that conversion to a utils class also. 
The `ImageFormatGroup` only applies to `CameraImage`, which is only returned from an ´imageStream´.

Reply:
On iOS the `videoFormat` is used for image streaming and for video recording. Meaning once we start supporting it, it needs to stay in there even when we refactor the streaming features.